src/ui/OmniboxResultList/OmniboxResultList.ts

```diff
@@ -306,6 +322,16 @@ export class OmniboxResultList extends ResultList implements IComponentBindings
     }
   }
 
+  private handleQuerySuggestSelected(args: IQuerySuggestSelected) {
+    this.executeQueryHover(args.suggestion);
```

Comment:
Maybe you should not call this method unnecessarily (if the option `numberOfPreviewResult` is not defined by the user)